blocks/components/editable/index.js

```diff
@@ -57,6 +59,28 @@ export default class Editable extends wp.element.Component {
 		this.props.onChange( value );
 	}
 
+	onKeyDown( event ) {
+		if ( ! this.props.tagName && event.keyCode === 13 ) {
+			// Wait for the event to propagate
+			setTimeout( () => {
+				// Getting the content before and after the cursor
+				this.editor.selection.getStart();
```

Comment:
The first point of my comment still applies for calling on `this.editor` in the context of `setTimeout`:

>Problem with a `setTimeout` is we can't be guaranteed that the component didn't unmount in the last tick, meaning due to the behavior of the function `ref`, `this.editor` could potentially be `null`.

https://github.com/WordPress/gutenberg/pull/409#discussion_r111174226